public/video-ui/src/components/FormFields/SelectBox.js

```diff
@@ -10,19 +10,11 @@ export default class SelectBox extends React.Component {
 
   renderDefaultOption = () => {
     if (this.props.displayDefault || !this.props.fieldValue) {
-      if (this.props.defaultOption) {
-        return (
-          <option value={this.props.defaultOption.id || null}>
-            {this.props.defaultOption.title || 'Please select...'}
-          </option>
-        );
-      } else {
-        return (
-          <option value={null}>
-            {'Please select...'}
-          </option>
-        );
-      }
+      return (
```

Comment:
Is there a reason we want to remove this code?

Reply:
Discussed IRL: introduced in the last PR for this fix, so effectively a revert 👍

Reply:
ftr - because i added it last time, but is now no longer needed